package.json

```diff
@@ -54,5 +54,8 @@
       "vue-cli-service lint",
       "git add"
     ]
+  },
+  "resolutions": {
```

Comment:
What's this for?

Reply:
To force a version of the serialize-javascript library that had a high vulnerability. This was fixed the day after I committed these changes lol https://github.com/webpack-contrib/copy-webpack-plugin/releases/tag/v5.1.2